libs/wordpress-plugin/plugin/trunk/block-protocol.php

```diff
@@ -212,6 +212,22 @@ function block_protocol_init()
     return;
   }
 
+	// register and enqueue the sentry plugin if plugin analytics are enabled
```

Comment:
We should probably move this into the `block_protocol_editor_assets` function below because that's where the other client-side JS gets loaded – we don't need to load Sentry on non-editor pages (e.g. settings page) because we don't load any `blockprotocol` JS there. It can also go below the `get_block_protocol_blocks` check because they won't even be able to select blocks if it's failed

Reply:
Trying to test this change, but seems Sentry is being a little slow right now